.github/workflows/nightly_detect_failure.yml

```diff
@@ -178,9 +177,12 @@ jobs:
         CURRENT_VERSION: ${{ steps.current_version.outputs.current_version }}
         BUMP_VERSION: ${{ steps.bump_version.outputs.result }}
         BUMP_BRANCH: ${{ steps.latest_bump_branch.outputs.result }}
+        ZULIP_API_KEY: ${{ secrets.ZULIP_API_KEY }}
       shell: python
       run: |
         import os
+        import zulip
+        client = zulip.Client(email='github-mathlib4-bot@leanprover.zulipchat.com', api_key=os.getenv('ZULIP_API_KEY'), site='https://leanprover.zulipchat.com')
```

Comment:
How did this work at all without these lines?

Reply:
It didn't!